### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by cloud007bond**
- Added index.html
- Styles.css
- Script.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,16 +9,27 @@
...
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+         <ul class="desktop-nav">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
          </ul>
          <a href="#">
             <img src="./images/hamburger.svg" alt="hamburger menu" class="hamburger-img" id="hamburger">
```
Comment (stanleySimeon) [inline index.html:35]: According to the Figma template, I suggest you stick to the design as much as possible (e.g., font, colours, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), currently your popup mobile menu and positioning do not match the required design.

> ## Your Design
>
>Screenshot 2023-06-26 at 11 49 40 AM
>
> ## The Figma Design
>
>Screenshot 2023-06-26 at 12 03 48 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @cloud007bond, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good PR Description
- Meaningful commit messages
- Linters check have passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -9,16 +9,27 @@
...
 
       <nav>
-         <ul class="nav">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+         <ul class="desktop-nav">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
```
Comment (Shedrack-Sunday) [inline index.html:32]: - [ ] Kindly revisit the learning materials for this section to help you fix the pending issues. 
- I can see that the popup does not cover the whole screen just as it is on the template you chose, and as suggested by the last reviewer.  This [Link](https://dev.to/javascriptacademy/responsive-navigation-bar-with-mobile-menu-using-html-css-2hp) might come in handy to get a heads up 

- Kindly adjust the left margin space in your design in relation to the template prrovided. [This detailed Link](https://www.freecodecamp.org/news/how-to-build-a-responsive-navigation-bar-with-dropdown-menu-using-javascript/) has resourse that might come in handy 👍 

### Your design:
image

### Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @cloud007bond      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the setup✔️  
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @mulugithub     👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the use of GitHub features✔️ 
- Kudos on implementing the features as requested ✅

### Verified view:
image
 
## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

- [ ] The styling of your cards seems a little bit scattered now. Kindly take time to adjust the styling to fit the design given 👍 

Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**